pyspider/database/elasticsearch/projectdb.py

```diff
@@ -28,7 +28,9 @@ def __init__(self, hosts, index='pyspider'):
                 }
             })
 
-    def insert(self, name, obj={}):
+    def insert(self, name, obj=None):
```

Comment:
The reason of "Use None and Docstrings to Specify Dynamic Default Arguments" is the argument default dict is created when defined, it may be modified afterward.
But here, the `obj` dict will always been copied, there is no way to modify the default dict, so there is no need to create it again.